Fix issue 1644 (#1646)
uyuni-project/uyuni-docs: modules/client-configuration/pages/contact-methods-saltbundle.adoc

Using the Salt Bundle, it is no longer needed that the client provides any Python interpreter or modules.

> Before bootstrapping new client you need to include [package]``venv-salt-minion`` to the bootstrap repository relevant to the client with explicitly specifying [package]``venv-salt-minion`` as an additionall package on calling [command]``mgr-create-bootstrap-repo``.

Reviewer: Drive-by edit:
```suggestion
Before bootstrapping new client you need to include [package]``venv-salt-minion`` to the bootstrap repository relevant to the client with explicitly specifying [package]``venv-salt-minion`` as an additional package on calling [command]``mgr-create-bootstrap-repo``.
```
[I think we can break this sentence into two short sentences later.]

[IMPORTANT]
====
{productname} 4.2 is not using the Salt Bundle to for Salt SSH (including bootstrapping with {webui}) by-default.

> To enable the Salt Bundle with Salt SSH support the following parameter needs to be specified in [path]``/etc/rhn/rhn.conf``:
> ----
> web.ssh_salt_pre_flight_script = /usr/share/susemanager/salt-ssh/preflight.sh
> ----
> To prevent using salt thin on managing Salt SSH actions the following parameter needs to be specified in [path]``/etc/rhn/rhn.conf``:
> ----
> web.ssh_use_salt_thin = false
> ----

Reviewer: I'd collapse both these params:

```suggestion
To enable the Salt Bundle with Salt SSH support and to disable so-called Salt thin, specify in [path]``/etc/rhn/rhn.conf``:
----
web.ssh_salt_pre_flight_script = /usr/share/susemanager/salt-ssh/preflight.sh

web.ssh_use_salt_thin = false
----
```
I'm not sure whether we must explain "Salt thin".  I can only guess what this is ;-)
